From 59bfacd92f438ee2ed50e3c35a7bd74d2a5a122d Mon Sep 17 00:00:00 2001
From: "yang.wang" <yang.wang@deepcomputing.io>
Date: Thu, 13 Mar 2025 18:10:47 +0800
Subject: [PATCH 232/534] Kconfig: fix compilation warnings.
 drivers/soc/eswin/ai_driver/Kconfig:21:warning: no new line at end of file

Signed-off-by: yang.wang <yang.wang@deepcomputing.io>
---
 drivers/soc/eswin/ai_driver/Kconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/soc/eswin/ai_driver/Kconfig b/drivers/soc/eswin/ai_driver/Kconfig
index 0044a4be10e5e..e2d60db9a5e2a 100644
--- a/drivers/soc/eswin/ai_driver/Kconfig
+++ b/drivers/soc/eswin/ai_driver/Kconfig
@@ -18,4 +18,5 @@ config ESWIN_NPU
 config NPU_PERF
     bool "NPU PERF"
     default n
-    depends on ESWIN_NPU
\ No newline at end of file
+    depends on ESWIN_NPU
+
-- 
2.53.0

